src/utils/network/session-core.js

```diff
@@ -507,8 +509,8 @@ export const sendMessageTTL = async (session, messageObject) => {
 
   try {
     await Promise.all([
-      await saveMessageTTLInInbox(session, containerUrl, newSolidDataset, messageSlug),
-      await saveMessageTTLInInbox(session, inboxUrl, newSolidDataset, messageSlug)
+      await saveMessageTTL(session, containerUrl, newSolidDataset, messageSlug),
```

Comment:
Are the `awaits` here necessary?

Reply:
Yeah, it's needed for the async function that saveMessageTTL uses underneath.

Reply:
No, I just tested this, and the `awaits` on lines 493 and 494 are unnecessary, and seem to slow the code down. These lines should be:
```
await Promise.all([
  saveMessageTTL(session, containerUrl, newSolidDataset, messageSlug),
  saveMessageTTL(session, outboxUrl, newSolidDataset, messageSlug)
])

```
You already handle the situation correctly on line 492. "Send out this list of promises, and wait until all of them return." These extra awaits render the `Promise.all` irrelevant. It forces the code to wait for the first `saveMessageTTL` to resolve before it sends the second request, which isn't the behavior we want.

Reply:
Got it, I see what you mean. The await have already been handled. With the await outside the Promise.all

Fixing this.